Cubical/Codata/M/AsLimit/M/Base.agda

```diff
@@ -151,7 +149,7 @@ shift-iso S@(A , B) =
       α-iso-step-5-Iso =
         Σ-cong-iso (lemma11-Iso {S = S} (λ _ → A) (λ _ x → x)) (λ a,p →
           Σ-cong-iso (pathToIso (cong (λ k → (n : ℕ) → k n) (funExt λ n → cong (λ k → B k → Wₙ S n) (α-iso-step-5-Iso-helper0 (a,p .fst) (a,p .snd) n)))) λ u →
-                              pathToIso (cong (λ k → (n : ℕ) → k n) (funExt λ n → isoToPath (α-iso-step-5-Iso-helper1-Iso (a,p .fst) (a,p .snd) u n))))
+                              pathToIso (cong (λ k → (n : ℕ) → k n) (funExt λ n → α-iso-step-5-Iso-helper1-Iso (a,p .fst) (a,p .snd) u n)))
```

Comment:
For example these uses of pathToIso could probably be removed. In fact I don't know if you should express this as an Iso at all. It's often easier to work with paths as much as possible

Reply:
I tried writing this as with paths however it was slower than what it is currently for some reason.

Reply:
I see. I guess pathToIso is actually not too bad. The problem is the other way around as it uses univalence which tends to lead to very big terms if you unfold things.